src/askscript/__tests__/04-control-flow/program10d_2b-if_else_if_else.ast.tsx

```diff
@@ -0,0 +1,46 @@
+export = (
```

Comment:
I think there should be much more files added than just one .tsx file.
Look what I got after running `npm run build && npm test`:

![image](https://user-images.githubusercontent.com/12752520/93124123-6e714d80-f6c9-11ea-8da4-f1bf42dba386.png)


All these files should be added to your commit.


Also, your changes to the code resulted in changes in the existing .ast.tsx files. You should commit them too:
![image](https://user-images.githubusercontent.com/12752520/93124222-9365c080-f6c9-11ea-9c76-b0bb8695edcc.png)

Reply:
Indeed, all missing *.askcode and *.ast.tsx have been pushed.

I am using git client sourcetree which does makes it easier identify, stage, and commit untracked files.

Let me know if anything is still missing.